df_py/predictoor/calc_rewards.py

```diff
@@ -40,21 +40,44 @@ def calc_predictoor_rewards(
         The number of tokens available for distribution as rewards.
 
     @return
-    rewards -- dict of [pdr_address] : float
-        The calculated rewards for each predictoor.
+    rewards -- dict of [contract addr][predictoor addr]: reward
+        The calculated rewards for each predictoor per contract address.
     """
 
+    predictoor_contracts = query_predictoor_contracts(chain_id)
+    tokens_per_contract = tokens_avail / len(predictoor_contracts)
```

Comment:
I see tokens_avail is Union[int, float]. If it is an int, this division will be rounded. Can you cast the type here, if possible?